.markdownlint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "default": true,
+    "MD004": false,
+    "MD007": false,
+    "MD013": {
+        "line_length": 1000
+    },
+    "MD024": false,
+    "MD032": false
+}
```

Comment:
I'm not totally sure about this list (MD032 for example looks maybe should be enabled, but haven't tested yet). Also, if this config is here but not being run in CI, even if we fix things right now with it, it'll get degraded over time. Can you add something to the CI to run the lint?

Reply:
Sure I can add a CI check for it, I disabled all that was the actual style in the file
My proposal is to add a CI check and after merge fix whitespaces and the Markdown lint issues, sounds good ?